### ISSUE TITLE
Center of VideoNode seems to be incorrect

### ISSUE THREAD

**Issue by engor** [SceneView/sceneform-android#214]
Hi. When I create VideoNode it shows with top-right offset, I looked at the code:
```
    public Renderable makePlane(float width, float height, Material material) {
        return PlaneFactory.makePlane(
                new Vector3(width, height, 0.0f),
                new Vector3(width, height, 0.0f),
                material
        );
    }
 ```
the second parameter `new Vector3(width, height, 0.0f)` is plane center, so that center of plane shifted by size of a video.

It's necessary to have center right in the real center of video to be able to rotate node around it.

**Comment by RGregat** [comment]
Thx, It will be solved with this PR https://github.com/SceneView/sceneform-android/pull/217

**Comment by engor** [comment]
Thank you! Hope it will be merged soon. :)

**Comment by ThomasGorisse** [comment]
Fixed in #2386c772f9ca9a9e0b5c9cc1f64606d9acee8da4
